independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiDefinition.swagger.json

```diff
@@ -0,0 +1,201 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Default title",
+    "description": "This is a connector to the Open Trivia Database to allows access to it's fun trivia questions",

```

Comment:
@Kiveshan, connector description should end with a period. And please make sure that all files are free of grammatical errors. here you have "to allows" and I think you meant to say, "to allow".